DataProcessing/datax-host/src/main/scala/datax/input/BlobPointerInput.scala

```diff
@@ -39,7 +39,7 @@ object BlobPointerInput {
     (new StructType).add("BlobPath", StringType)
   }
 
-  private val saRegex = s"""wasbs?://[\w-]+@([\w\d]+)\${BlobProperties.BlobHostPath}/.*""".r
+  private val saRegex = s"""(wasbs?|abfs?)://[a-zA-Z0-9-_]+@([a-zA-Z0-9-_]+)(${BlobProperties.BlobHostPath}||${BlobProperties.DfsHostPath})/.*""".r
```

Comment:
I had to change the regex here since I was getting an error using \w. I want to be sure that this doesn't break any existing functionality.

Reply:
I was working in a change on that exact line in another PR, however i prefer your fix over mine. I think it would be a good idea to add a unit test just to ensure \w error is gone. Can you add a very basic BlobPointerInput test to make sure the class is initialized without exceptions, take as example the unit test added at https://github.com/microsoft/data-accelerator/pull/192/files , then i can remove the changes on this file/unit test from my PR